static/js/partners.js

```diff
@@ -1,35 +1,44 @@
-/**
- * Generate a warnin only if warnings are available.
- */
-function warn(message) {
-    if (typeof(console) == 'object' && typeof(console.warn) == 'function') {
-        console.warn(message);
-    }
-}
 
 /**
- * Call the partners API and load
+ * Partners logo cloud
+ *
+ * @project        Ubuntu Core Front-End Framework
+ * @author         Web Team at Canonical Ltd
+ * @copyright      Canonical Ltd
  */
-YUI().use('jsonp', 'node', function(Y) {
-    dependencies = true;
 
-    if (typeof(partnerLogoClouds) != 'object') {
-        warn('Load partner logos: Expected array "partnerLogoClouds" not found. Stopping.');
-        dependencies = false;
-    }
+if (!core) { var core = {}; }
```

Comment:
This may not be how it was done in the past, but I would rather use namespaces to actually mean something useful rather than using one single `core` namespace. Thus, `core.js` simply states `var core = {svgFallback: function() {...}}` - completely owning the `core` namespace, and this file would similarly declare:

``` javascript
var partners = {
    loadPatnerClouds = function() {...}
}
```